go-controller/pkg/allocator/udn/allocator.go

```diff
@@ -19,6 +20,14 @@ func New(watchFactory factory.NodeWatchFactory, nodeName string) *Allocator {
 	}
 }
 
+func (a *Allocator) AllocateConntrackMark(networkName string) (uint, error) {
+	return a.calculateIDsFromNetwork("ct-mark", networkName, config.Gateway.UserDefinedNetworkConntrackMarkBase, config.Default.MaxUserDefinedNetworks, 1)
```

Comment:
I'd hard code `ct-mark` and `vrf-table` as constants in this file and use that const string value here